### ISSUE TITLE
hello， i found the 5g cell info is null

### ISSUE THREAD

**Issue by Bekeys** [eslamfaisal/FlutterCellInfo#14]
@eslamfaisal @MHarooney

**Comment by Bekeys** [comment]
@eslamfaisal

**Comment by darixsamani** [comment]
@Bekeys this user solved this issue on his branch

check this  [link](https://github.com/daviddagan/FlutterCellInfo/commit/d59ee86544e88e7197ff7f4afd80528ac2b60404)

**Comment by Bekeys** [comment]
Thanks， but I didn't find the specific branch  @darix15

**Comment by darixsamani** [comment]
recheck this [link](https://github.com/daviddagan/FlutterCellInfo/commit/d59ee86544e88e7197ff7f4afd80528ac2b60404)

**Comment by darixsamani** [comment]
you can directly use this repository : [link](https://github.com/daviddagan/FlutterCellInfo)

**Comment by Bekeys** [comment]
Yes I see the modification, but I don't think the source of the problem is here

final String version = await _channel.invokeMethod('cell_info');

version is null

**Comment by Bekeys** [comment]
@darix15
